radix-engine-tests/tests/recallable.rs

```diff
@@ -114,3 +115,79 @@ fn can_take_on_recallable_vault() {
         .unwrap();
     assert_eq!(other_amount, Decimal::one());
 }
+
+#[test]
+fn test_recall_on_internal_vault() {
```

Comment:
Can we update name to `owned_internal_vault`

Reply:
No problem.